.env.example

```
@@ -0,0 +1 @@
+REACT_APP_URL_MEMBERS = /members
```

Por las dudas es recomendable sacar los espacios y usar comillas para que no haya margen de error:

```suggestion
REACT_APP_URL_MEMBERS="/members"
```

---

Obviamente también cambiarlo en el .env